crates/circuit/src/operations.rs

```diff
@@ -1338,7 +1338,7 @@ impl Operation for StandardGate {
                             (Self::H, smallvec![], smallvec![Qubit(0)]),
                             (Self::Sdg, smallvec![], smallvec![Qubit(0)]),
                         ],
-                        FLOAT_ZERO,
+                        Param::Float(PI / 4.),
```

Comment:
This global phase (and the global phase for SXdg decomposition) were incorrect in the Rust code here, but correct both in the Python code and in the equivalence library. Does it make sense to submit just these fixes in a separate PR so that we could backport it? I guess the only way the incorrect could come up in practice is when calling HighLevelSynthesis (since it now uses Rust-based gate definitions) but without the standard equivalence library (since this beats definitions priority-wise).